freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientBuilder.java

```diff
@@ -180,14 +180,14 @@ public FreeIpaClient build(boolean withPing) throws URISyntaxException, IOExcept
             }
         }
         try {
-            CookieAndStickyId cookieAndStickyId = connect(user, pass, clientConfig.getApiAddress(), port, stickyIdHeader, stickyId);
+            CookieAndStickyId cookieAndStickyId = connect(user, pass, clientConfig.getApiAddress(), port, stickyId);
             String sessionCookie = cookieAndStickyId.getCookie();
             stickyId = cookieAndStickyId.getStickyId();
 
             Map<String, String> headers = buildHeaders(sessionCookie, stickyId);
 
             JsonRpcHttpClient jsonRpcHttpClient = new JsonRpcHttpClient(ObjectMapperBuilder.getObjectMapper(),
-                    getIpaUrl(clientConfig.getApiAddress(), port, basePath, "/session/json"), headers);
+                    getIpaUrl(clientConfig.getApiAddress(), port, basePath, "/session/json"), headers, false, true);
```

Comment:
is there any downside to compressing the requests as well?

Reply:
didn't bother to try out, I don't expect large request. compression is taking some CPU and memory but I don't think that should matter

Reply:
ok. requests are small compared to the response from user_find and group_find. We can revisit if the requests become a problem.